.gemini.yml

```diff
@@ -10,12 +10,28 @@ system:
 
 browsers:
   chrome:
+    windowSize: 1600x1000
     desiredCapabilities:
       browserName: "chrome"
       version: "66.0"
       platform: "Windows 10"
 
+  firefox-medium:
+    windowSize: 1000x600
+    desiredCapabilities:
+      browserName: "firefox"
+      version: "47.0"
+      platform: "Windows 10"
+
+  firefox-portrait:
+    windowSize: 500x800
+    desiredCapabilities:
+      browserName: "firefox"
+      version: "47.0"
+      platform: "Windows 10"
+
```

Comment:
Had to use Firefox, because Chrome wasn't responding well to the `windowSize` instruction.

Reply:
Is it possible to mock the parent constraints in the test instead of changing the gemini configuration? 🤔

Reply:
Moved resizing to the tests file.